idseq_dag/steps/generate_host_genome.py

```diff
@@ -141,7 +143,8 @@ def make_star_index(fasta_file, gtf_file, output_star_genome_path):
                 '--runThreadN',
                 str(multiprocessing.cpu_count()), '--runMode', 'genomeGenerate',
                 *gtf_command_part, '--genomeDir', star_genome_part_dir,
-                '--genomeFastaFiles', fasta_file_list[i]
+                '--genomeFastaFiles', fasta_file_list[i],
+                '--limitGenomeGenerateRAM', virtual_memory().available
```

Comment:
I would like some feedback on this. Personally, I think limiting the RAM just gets in the way, potentially making you need to restart the job after it has been running for some time while we run them on machines with ample RAM for the specific purpose of running these jobs. This will also keep RAM within limits provided more RAM isn't used up by other processes while it's running.

Reply:
I feel the same way since it has wasted my time as well.. seems safe enough for this type of command